.github/workflows/e2eEnvironment.yml

```diff
@@ -27,8 +27,9 @@ jobs:
   kind:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        version: [1.19, 1.20]
+        version: ["1.19", "1.20", "1.21", "1.22", "1.23", "1.24"]
```

Comment:
I think we can remove some of the older version here. 1.21 is the oldest version available for new GKE clusters, so I think we can remove 1.19 and 1.20.

Reply:
It’s not a speed issue since they run in parallel. So the only reason to limit it is cost, but I’m not sure if we’re paying GitHub for it or getting free actions for being open source.

Reply:
Yeah, we don't have any policy defined around the support k8s versions. We can keep them for now and adjust it in a separate PR by looking at k8s OSS and GKE version policy.